### PR TITLE
Prepare for follow-up release for November providers

### PR DESCRIPTION
**Pull request by potiuk**
Co-authored-by: Daniel Standish <15932138+dstandish@users.noreply.github.com>

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code changes, an Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvement+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragment file, named `{pr_number}.significant.rst` or `{issue_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

**Comment by potiuk** [comment]
I decided to run a "full" release of providers as follow-up - i.e. include all the small changes that were merged since 15th Nov - this way it will be "full" provider's release and documentation will be consistent.

**Comment by potiuk** [comment]
Replaces #27726

**Comment by potiuk** [comment]
I also skipped asana from previous wave and added it here as feature release.

File: airflow/providers/asana/CHANGELOG.rst [hunk 1]
```diff
@@ -43,14 +37,18 @@ Misc
 Features
 ~~~~~~~~
 
+In AsanaHook, non-prefixed extra fields are supported and are preferred.  So if you should update your
```
Comment (jedcunningham) [inline airflow/providers/asana/CHANGELOG.rst]:
```suggestion
In AsanaHook, non-prefixed extra fields are supported and are preferred. You should update your
```

**Review by jedcunningham — APPROVED** [review]

**Comment by Taragolis** [comment]
I'm noticed that providers refers it their pages to examples (system tests  and legacy examples) from earlier versions, e.g.:
- Amazon refers to branch 4.0.0: https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/example-dags.html
- Google refers to branches 8.0.0 to 8.3.0 https://airflow.apache.org/docs/apache-airflow-providers-google/stable/example-dags.html

I've also found the same situation with other providers. 

Seems like it not update automatically and we could correct links within this PR.

**Comment by potiuk** [comment]
> I'm noticed that providers refers it their pages to examples (system tests and legacy examples) from earlier versions, e.g.:
> 
> * Amazon refers to branch 4.0.0: https://airflow.apache.org/docs/apache-airflow-providers-amazon/stable/example-dags.html
> * Google refers to branches 8.0.0 to 8.3.0 https://airflow.apache.org/docs/apache-airflow-providers-google/stable/example-dags.html
> 
> I've also found the same situation with other providers.
> 

Ah. Interesting. Good catch. they **should** be automatically updating when the documentation is built, but maybe there is a bug in the process. I will take a look at that - maybe we will have to fix some historical links as well.

**Comment by potiuk** [comment]
OK. That one should be good to go with follow-up release of providers. 

We have a bit more changes than initially planned due to databricks/google/common.sql incompatibilities - with this one we seems to be back-on-track (at least with known issues).

**Comment by potiuk** [comment]
> I'm noticed that providers refers it their pages to examples (system tests and legacy examples) from earlier versions, e.g.:

Should be fixed be #28090

**Pull request merged**